src/main.rs

```diff
@@ -344,6 +346,10 @@ struct Args {
         hide = true
     )]
     help_markdown: bool,
+
+    #[clap(long = "http-admin")]
```

Comment:
will this bind the environment variable as well?

Reply:
only flag, let me add that quickly